src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -230,6 +233,11 @@ class SharedLinkSection extends React.Component<Props, State> {
             allowedPermissionLevels = allowedPermissionLevels.filter(level => level !== CAN_VIEW_DOWNLOAD);
         }
 
+        // if the user cannot edit, we remove this option from the dropdown
+        if (!isEditSettingAvailable || !isAllowEditSharedLinkForFileEnabled) {
```

Comment:
`isAllowEditSharedLinkForFileEnabled` is a FF that is passed down from the parent app.